src/smokeTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/JurisdictionAuthenticationTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.typesafe.config.ConfigFactory;
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.TestPropertySource;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.model.out.JurisdictionConfigurationStatus;
+import uk.gov.hmcts.reform.logging.appinsights.SyntheticHeaders;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@TestPropertySource("classpath:application.conf")
+public class JurisdictionAuthenticationTest {
+
+    private static final String TEST_URL = ConfigFactory.load().getString("test-url");
+
+    @DisplayName("Each configured jurisdiction should have valid credentials")
+    @Test
+    public void each_jurisdiction_should_have_valid_credentials() throws IOException {
+        String failMessage = "Misconfigured %s jurisdiction, error description: %s. Check the logs for more details";
+        byte[] response = RestAssured
+            .given()
+            .relaxedHTTPSValidation()
+            .baseUri(TEST_URL)
+            .header(SyntheticHeaders.SYNTHETIC_TEST_SOURCE, "Bulk Scan Orchestrator smoke test")
+            .get("/jurisdictions")
+            .andReturn()
+            .asByteArray();
+        ObjectMapper mapper = new ObjectMapper();
+
+        mapper.readTree(response).elements().forEachRemaining(responseStatus -> {
+            JurisdictionConfigurationStatus status = new JurisdictionConfigurationStatus(
+                responseStatus.get("jurisdiction").asText(),
+                responseStatus.get("is_correct").asBoolean(),
+                responseStatus.get("error_description").asText()
+            );
+
+            assertThat(status)
+                .withFailMessage(failMessage, status.jurisdiction, status.errorDescription)
+                .isEqualToComparingFieldByField(
```

Comment:
we're only interested in `is_correct`

Reply:
if it is incorrect, the failure message will be present in smoke test log - can potentially help out to speed track the failure instead of going to portal app insights figuring out the cause, here's an example:

```bash
java.lang.AssertionError: Misconfigured local jurisdiction, error description: Connection refused (Connection refused) executing POST http://localhost:8080/oauth2/authorize?response_type=code&client_id=bsp&redirect_uri=http%3A%2F%2Flocalhost%2Freceiver. Check the logs for more details
```

Reply:
that's the fail message configured in line above, right?
I'm saying that in assert we should check all items have `is_corrent == true`

Reply:
we do. along with jurisdiction. it's no biggy nor complicated. not sure what you are trying to achieve here as test is pretty obvious and straight-forward

Reply:
it's checking the jurisdiction by comparing `status.jurisdiction` to `status.jurisdiction` though